src/tss2-esys/esys_tr.c

```diff
@@ -221,7 +222,7 @@ Esys_TR_FromTPMPublic_Finish(ESYS_CONTEXT * esys_context, ESYS_TR * object)
         objectHandleNode->rsrc.misc.rsrc_key_pub = *public;
         SAFE_FREE(public);
         SAFE_FREE(name);
-            SAFE_FREE(qualifiedName);
+        SAFE_FREE(qualifiedName);
```

Comment:
This change seems to be unrelated to the issue identified in the commit message. Is this just a whitespace fix?

Reply:
Yes, I remove this from the commit.